githublib.pl

```diff
@@ -24,4 +24,8 @@
     format(atom(Path), '/repos/~w/~w/pulls', [Owner, Project]),
     github_json(Path, Dict).
 
+github_issues(Owner, Project, Dict) :-
+    format(atom(Path), '/repos/~w/~w/issues?state=all&access_token=e427390592197574257a31f6e8a81cad8697df41', [Owner, Project]),
```

Comment:
what is this token?

Reply:
Nothing, act like you've never seen it :)